airflow/jobs.py

```diff
@@ -403,39 +422,77 @@ def schedule_dag(self, dag):
                     dr.end_date = datetime.now()
             session.commit()
 
+            # check last scheduled run
+            # get this dag's max execution date by
+            #  external trigger is false
+            #  and "like" Dag Run ID.ID
+            # bolke: why a like?
```

Comment:
There are 3 different prefixes currently : trigger, manual, and scheduled. This is looking for scheduled dag runs.